spec/lib/bank_accounts/account_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'spec_helper'
+
```

Comment:
Rspec's magic also makes it such that you shouldn't have to `require 'spec_helper'` here--it show know which tests to run.